tests/clone_test.go

```diff
@@ -0,0 +1,54 @@
+package tests_test
+
+import (
+	"flag"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	k8sv1 "k8s.io/api/core/v1"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/containerized-data-importer/tests"
+)
+
+const (
+	TestNS  = "default"
+	TestPOD = "test-pod"
+)
+
+var _ = Describe("Cloning", func() {
+	var (
+		pvc *k8sv1.PersistentVolumeClaim
+	)
+
+	flag.Parse()
+	client, err := tests.GetKubeClient()
+	tests.PanicOnError(err)
```

Comment:
Given this uses the ginkgo framework, it's better to use `Expect(err).NotTo(HaveOccurred)` which  can take a 2nd string arg for a custom message.  The 2nd benefit being that it will report in a ginkgo defined format.